stackslib/src/chainstate/stacks/auth.rs

```diff
@@ -1387,6 +1388,35 @@ impl TransactionAuth {
             }
         }
     }
+
+    pub fn is_supported_in_epoch(
```

Comment:
Thanks for this.

1. Can you add a docstring?
2. Can you add a dedicated unit test for it?

Reply:
added @jcnelson